torch_xla/csrc/xla_lower_util.cpp

```diff
@@ -266,6 +266,31 @@ xla::XlaOp BuildDropout(const xla::XlaOp& input, float probability) {
   return input * mask;
 }
 
+xla::XlaOp BuildRandperm(xla::int64 n, xla::PrimitiveType element_type,
+                         xla::XlaBuilder* builder) {
+  xla::XlaOp input = xla::Iota(builder, element_type, n);
+
+  // Ensure that the key space is greater than or equal to the cube of the
+  // number of values. Inspired by RandomShuffleOp in tf2xla, where the full
+  // rationale for picking the exponent value is described.
+  constexpr int exponent = 3;
+  const int rounds = static_cast<int>(
+      std::ceil(exponent * std::log(n) / std::log(tensorflow::kuint32max)));
+
```

Comment:
Remove some vertical space in this function.

Reply:
Done.